### PR TITLE
Rename example file for zipkin as appropriate

### LINKED ISSUE #187: Improve documentation
* Change name for Zipkin example file

The name for the Zipkin example file currently reads AlwaysSampleTraceExample.php instead of AlwaysOnTraceExample.php this can be a little confusing if you try searching for the former with your codebase

* Add Jaeger example file to the readme

Currently, there is no pointer for the Jaegar example file on the readme, even though Jaeger features as one of the example implementations of this project, I think it might be useful to add a pointer to the example file

### PR DESCRIPTION
**Pull request by prondubuisi**
Mention Jaegar example file in readme

fixes #187

### CODE REVIEW

**Review by bobstrecansky — COMMENTED** [review]
I think this is a good improvement.  

- Wondering if it'd be clearer to rename `make examples` to `make trace examples`?
- Should we potentially call these `AlwaysOnZipkinExample` and `AlwaysOnJaegerExample` to make them a bit clearer?

**Comment by prondubuisi** [comment]
> I think this is a good improvement.
> 
> * Wondering if it'd be clearer to rename `make examples` to `make trace examples`?
> * Should we potentially call these `AlwaysOnZipkinExample` and `AlwaysOnJaegerExample` to make them a bit clearer?

Nice pointers @bobstrecansky 

Point 1 will make what we are doing clearer, so that should work
Point 2 will be very nice too for consistency 

I think this PR will have to wait until I make PRs to reflect point 1 and 2, I will then update this PR to reflect the changes,  thanks

**Comment by bobstrecansky** [comment]
You can also make those updates in this PR if you'd like @prondubuisi !

**Comment by prondubuisi** [comment]
I prefer keeping them separate, Makes it easier for me to keep track, and implement changes when needed.

File: README.md [hunk 1]
```diff
@@ -56,9 +56,11 @@ Execute `make test` from your bash compatible shell.  This will output the test
 ## Examples
 
 ### Trace
-You can use the [examples/AlwaysSampleTraceExample.php](/examples/AlwaysOnTraceExample.php) file to test out the reference implementation we have.  This example perfoms a sample trace with a grouping of 5 spans and POSTs the result to a local zipkin instance.
+You can use the [examples/AlwaysOnTraceExample.php](/examples/AlwaysOnTraceExample.php) file to test out the reference implementation we have for zipkin.  This example perfoms a sample trace with a grouping of 5 spans and POSTs the result to a local zipkin instance.
 
-The PHP should execute by itself (if you have a zipkin instance running on localhost), but if you'd like a no-fuss way to test this out with docker and docker-compose, you can perform the following simple steps:
+You can also use the [examples/JaegerExporterExample.php](/examples/JaegerExporterExample.php) file to test out the reference implementation we have for jaegar.  This example perfoms a sample trace with a grouping of 5 spans and POSTs the result to a local jaegar instance.
```
Comment (bobstrecansky) [inline README.md:61]: Should we be renaming this file to `AlwaysOnJaegerTraceExample`?

File: README.md [hunk 2]
```diff
@@ -56,9 +56,11 @@ Execute `make test` from your bash compatible shell.  This will output the test
 ## Examples
 
 ### Trace
-You can use the [examples/AlwaysSampleTraceExample.php](/examples/AlwaysOnTraceExample.php) file to test out the reference implementation we have.  This example perfoms a sample trace with a grouping of 5 spans and POSTs the result to a local zipkin instance.
+You can use the [examples/AlwaysOnTraceExample.php](/examples/AlwaysOnTraceExample.php) file to test out the reference implementation we have for zipkin.  This example perfoms a sample trace with a grouping of 5 spans and POSTs the result to a local zipkin instance.
```
Comment (bobstrecansky) [inline README.md:59]: Should we be renaming this file to `AlwaysOnZipkinTraceExample`?

Comment (prondubuisi) [reply README.md:59]: Thanks for the pointers, updating this PR now

Comment (prondubuisi) [reply README.md]: Updated @bobstrecansky

**Review by bobstrecansky — APPROVED** [review]

**Pull request merged**